examples/kosmos-2-patch14-224/README.md

```diff
@@ -0,0 +1 @@
+Image generation and Entity extraction using the microsoft/kosmos-2-patch14-224 model from Microsoft
```

Comment:
Maybe we can just name the model as `kosmos2`? And a title and some links to its paper and repo would be nice.